geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/membership/gms/locator/GMSLocatorIntegrationTest.java

```diff
@@ -53,14 +53,14 @@ public void viewFileIsNullByDefault() {
 
   @Test
   public void initDefinesViewFileInSpecifiedDirectory() {
-    gmsLocator.init(tcpServer);
+    gmsLocator.init("" + tcpServer.getPort());
```

Comment:
I've got feedback from the Community that `String.valueOf(int)` is preferred over empty quotes concatenation...